TFT/src/User/API/menu.h

```diff
@@ -198,13 +199,11 @@ void showLiveInfo(uint8_t index, const LIVE_INFO * liveicon, bool redrawIcon);
 void displayExhibitHeader(const char * titleStr, const char * unitStr);
 void displayExhibitValue(const char * valueStr);
 
-KEY_VALUES menuKeyGetValue(void);
+KEY_VALUES menuKeyGetValue(bool returnLongPressed);
 
 // smart home
 #ifdef SMART_HOME
```

Comment:
if the SMART_HOME is now always supported, remove this `#ifdef` from the entire project

Reply:
@digant73 Thanks a lot for the feedback! I just pushed an update reflecting your input.

About the SMART_HOME thing, it's still selectable, my changes don't affect this.
But... I see no real reason why one would want to disable it, so it could be removed if you ask me.
If you agree, then I can take out the Smart Home option everywhere needed.

BTW: The current status should be made into a release... it's a stable usable base. The last release is from 2021, which is very outdated!

Reply:
hhmm no, if still selectable, I would avoid to remove SMART_HOME (leave it as is)